test/DurableTask.AzureStorage.Tests/AzureStorageScenarioTests.cs

```diff
@@ -2312,6 +2314,40 @@ public async Task ScheduledStart_Activity_GetStatus_Returns_ScheduledStart(bool
             }
         }
 
+        /// <summary>
+        /// Test queue message generated by DTFx.AzureStorage v2.x can be successfully read by DTFx.AzureStorage v1.x
+        /// </summary>
+        [TestMethod]
+        public async Task TestBackwardCompatibilityWithASv2()
```

Comment:
Here you're testing that backwards compatibility is possible with the previous version, so I believe it should be `v1`
```suggestion
        public async Task TestBackwardCompatibilityWithASv1()
```